src/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+           
+           <header class="header">
+            <div class="header welcome">welcome</div>
+            <div class="header menu_icon"><img src="../media/icons/Union.png" alt="menu icon"/></div>
+           </header>
+           
+           <section class="home">
+                
+                        <div class="headlines">
+                            <h1 class="headline-1">Hey there. I’m Souad</h1>
+                            <h1 class="headline-2">I’m a software developer</h1>
+                           <!--   
+                            <div >   
+                            Hey there. I’m Souad
+                            </div>
+                            <div >
+                                I’m a software developer
+                            </div>-->
+                            
+                        </div>
+                        <div class="headline-description">
+                            <p>
+                                 I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                                don’t hestiate to contact me.
+                            </p>
+                        </div>
+                
+                <div class="social_media">
+                    <ul class="links">
+                        <li> <a><img src="../media/icons/github.png" alt="github"/></a></li> 
+                        <li><a><img src="../media/icons/twitter.png" alt="twiter"/></a></li>
+                        <li><a><img src="../media/icons/Linkedin.png" alt="linkdin"/></a></li>
+                        <li><a><img src="../media/icons/angellist.png" alt="angellist"/></a></li>
+                    </ul>    
+                </div>
```

Comment:
- [ ] [REQUIRED] It is nice to see that you have embedded the icons in an unordered and ordered list, however, the link to the icons are broken as they do not display on your page. Please refer to the design images above for proper clarity,